src/types/accountBillingStatus.js

```diff
@@ -7,13 +7,15 @@ class AccountBillingStatus {
     isTrial,
     trialPeriodEnds,
     isLegacy,
-    isGoodStanding = null
+    isGoodStanding = null,
+    isFree
```

Comment:
`isFree` should either be before `isGoodStanding` or be given a default value itself. not sure why `isGoodStanding` defaults to `null`, but someone made that choice on purpose and we should respect that. ie. we should be able to set `isFree` without knowing `isGoodStanding`.

i'd recommend just defaulting `isFree` to `null` as well. it was likely done for historical reasons to distinguish it being unset from it being false